File: StrykerDG.FarmForge.Api/Controllers/DTO/Requests/NewCropDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StrykerDG.FarmForge.LocalApi.Controllers.DTO.Requests
{
    public class NewCropDTO
    {
        public int CropTypeId { get; set; }
        public int VarietyId { get; set; }
        public int LocationId { get; set; }
        public int Quantity { get; set; }
        public DateTime Date { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/dto/new_crop_dto.dart
class NewCropDTO {
  int cropTypeId;
  int varietyId;
  int locationId;
  int quantity;
  DateTime date;

  NewCropDTO({
    this.cropTypeId,
    this.varietyId,
    this.locationId,
    this.quantity,
    this.date
  });
}